src/dynsight/_internal/analysis/entropy.py

```diff
@@ -54,6 +54,8 @@ def compute_data_entropy(
 
             assert np.isclose(data_entropy, 0.9993954419344714)
     """
+    if data.size == 0:
+        return 0.0
```

Comment:
Raise an error or return 0.0? To me, the size being 0 could be a mistake and should be handled by the user?